lib/auth.js

```diff
@@ -66,19 +68,24 @@ module.exports = new Model({
     this._bearerToken = response.access_token;
     apiClient.headers.Authorization = 'Bearer ' + this._bearerToken;
 
-    var refresh = this._refreshBearerToken.bind(this, response.refresh_token);
-    var timeToRefresh = (response.expires_in * 1000) - TOKEN_EXPIRATION_ALLOWANCE;
-    this._bearerRefreshTimeout = setTimeout(refresh, timeToRefresh);
+    this._bearerTokenExpiration = Date.now() + (response.expires_in * 1000);
```

Comment:
I think the way I've got it means the expiration time is relative to the user's clock, and this would make it relative to the server's clock. Isn't that a little safer?